### PR TITLE
Feat : Button Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**

### CODE REVIEW

File: assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -292,4 +292,99 @@ main {
...
+  background-color:blueviolet;
+  color: white;
+}
+.button--secondary{
+  margin: 8px;
+  background-color: transparent;
+  border: 2px solid blueviolet;
+  color:blueviolet;
+}
+.button--primary:hover{
```
Comment (dikshadhote) [inline assets/Css/ComponentsStyles.css:321]: Use cursor: pointer on button hover

**Comment by dikshadhote** [comment]
> _No description provided._

- Use some light background color instead of dark. All attention is going to background color instead of content
- Use cursor: pointer to show that button is clickable.
- Please don't use separate html files for every component, instead you can use one docs.html and put all components in it and make separate css files for each component

**Comment by ShreyashMahajan** [comment]
code looks nice

**Comment by SaurabhChawda** [comment]
> > _No description provided._
> 
> * Use some light background color instead of dark. All attention is going to background color instead of content
> * Use cursor: pointer to show that button is clickable.
> * Please don't use separate html files for every component, instead you can use one docs.html and put all components in it and make separate css files for each component

Okay, I will change the background color and also update the cursor as well.

**Comment by SaurabhChawda** [comment]
> > _No description provided._
> 
> * Use some light background color instead of dark. All attention is going to background color instead of content
> * Use cursor: pointer to show that button is clickable.
> * Please don't use separate html files for every component, instead you can use one docs.html and put all components in it and make separate css files for each component

ThankYou

**Comment by SaurabhChawda** [comment]
> code looks nice

ThankYou

**Pull request merged**